packages/web-ui/src/lab/Link/Link.docs.mdx

```diff
@@ -0,0 +1,128 @@
+import { Meta, Canvas, ArgTypes, Description } from '@storybook/blocks';
+import * as Stories from './Link.stories';
+
+<Meta of={Stories} />
+
+# Link
+
+<Description />
+
+<Canvas of={Stories.SimpleExample} />
+
+## Usage
+
+This `Link` component is part of the lab submodule, a way for us to make new
+components available prior to an official major release. This helps us bed in
+new components before introducing breaking changes, and to avoid blocking the
+availability of these components for designers.
+
+There are 2 options for importing this component.
+
+1. Import directly from the `lab` directory.
+
+```tsx
+import { Box } from '@utilitywarehouse/web-ui';
+import { Link } from '@utilitywarehouse/web-ui/dist/lab';
+
+[...]
+
+<Box>
+  <Link>Link</Link>
+</Box>
+```
+
+2. Import `Lab` directly from Web UI.
+
+```tsx
+import { Box, Lab } from '@utilitywarehouse/web-ui';
+
+[...]
+
+<Box>
+  <Lab.Link>Link</Lab.Link>
+</Box>
+// or
+const { Link } = Lab
+<Box>
+  <Link>Link</Link>
+</Box>
+```
+
+## Size
+
+The size prop controls the size of the `Link`. This is a responsive prop and
+can be used to display different sizes at different breakpoints.
+
+<Canvas of={Stories.ResponsiveSize} />
+
+```tsx
+<Link
+  size={{
+    mobile: 'small',
+    desktop: 'large',
+  }}
+/>
+```
+
+## With icons
+
+You can nest icons directly inside the `Link`. An appropriate gap is provided
+automatically, and the icon will inherit the appropriate colours.
+
+This Link is intended to be used with the UW icons packages. Small links
```

Comment:
it would be nice to include a link to the icons lib, it can help new joiners get familiar with the ecosystem